pkgs/servers/http/apache-modules/mod_pkcs12/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl, pkgconfig, mod_ca, apr, aprutil }:
+
+stdenv.mkDerivation rec {
+  pname = "mod_pkcs12";
+  version = "0.2.1";
+
+  src = fetchurl {
+    url = "https://redwax.eu/dist/rs/${pname}-${version}.tar.gz";
+    sha256 = "0by4qfjs3a8q0amzwazfq8ii6ydv36v2mjga0jzc9i6xyl4rs6ai";
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs =[mod_ca apr aprutil];
```

Consistent spacing is always nice:
```suggestion
  buildInputs = [ mod_ca apr aprutil ];
```

---

fixed (wondering if there is a better indent.pro).

---

You can try nixpkgs-fmt.